resources/views/pages/add-user.blade.php

```diff
@@ -0,0 +1,46 @@
+@extends('layout.main_layout')
+
+@section('addUser_content')
```

Comment:
If we only have one @yield('content') it would be easy for use to reuse templates